exercises/acronym/.meta/src/reference/java/Acronym.java

```diff
@@ -15,7 +15,8 @@ String getAcronym() {
     }
 
     private String generateAcronym(String phrase) {
-        return Arrays.stream(phrase.split("(?!')\\W"))
+        String newPhrase = phrase.replace("_", "");
```

Comment:
This could have a more descriptive name, e.g. `phraseWithoutUnderscores`? 🙂

Reply:
Great suggestion, I will change the name.